src/main/java/net/neoforged/neoforge/common/NeoForgeMod.java

```diff
@@ -161,24 +187,24 @@ public class NeoForgeMod {
      * @see IPlayerExtension#canReach(Entity, double)
      * @see IPlayerExtension#canReach(Vec3, double)
      */
-    public static final RegistryObject<Attribute> ENTITY_REACH = ATTRIBUTES.register("entity_reach", () -> new RangedAttribute("neoforge.entity_reach", 3.0D, 0.0D, 1024.0D).setSyncable(true));
+    public static final DeferredHolder<Attribute, Attribute> ENTITY_REACH = ATTRIBUTES.register("entity_reach", () -> new RangedAttribute("neoforge.entity_reach", 3.0D, 0.0D, 1024.0D).setSyncable(true));
 
     /**
      * Step Height Addition modifies the amount of blocks an entity may walk up without jumping.
      * 
      * @see IEntityExtension#getStepHeight()
      */
-    public static final RegistryObject<Attribute> STEP_HEIGHT = ATTRIBUTES.register("step_height", () -> new RangedAttribute("neoforge.step_height", 0.0D, -512.0D, 512.0D).setSyncable(true));
+    public static final DeferredHolder<Attribute, Attribute> STEP_HEIGHT = ATTRIBUTES.register("step_height", () -> new RangedAttribute("neoforge.step_height", 0.0D, -512.0D, 512.0D).setSyncable(true));
```

Comment:
We should erase to `Holder<R>` where applicable, especially if the declaration would be `DH<R, R>`

Reply:
That's what I was referring to [here](https://github.com/neoforged/NeoForge/pull/257#pullrequestreview-1723789927)